bitrise.yml

```diff
@@ -65,6 +48,7 @@ workflows:
         inputs:
         - test_to_run: Multiplatform.UItest.Tests(Android)
         - build_tool: msbuild
+        - emulator_serial: "emulator-5554"
```

Comment:
do we need this? the avd manager exposes the emulator serial and that env should be the default value of this input